assets/scripts/databaseAPI.js

```diff
@@ -65,7 +65,29 @@ class DatabaseAPI {
           description: "Mon site Scribouilli",
         }),
       }
-    )
+    );
+
+    this.createTopicGithubRepository(login, newRepoName);
+
+    return res;
```

Comment:
ce `return` a lieu temporellement avant que l'appel à `this.createTopicGithubRepository` ne soit terminé

D'un côté, c'est ptèt pas grave (parce qu'on n'a pas besoin d'attendre d'ajouter le topic pour commencer à rajouter/modifier des pages articles)
D'un autre, chuis pas à l'aise à ce que ça soit implicite
Est-ce que tu peux rajouter un commentaire pour le rendre explicite ?

Reply:
J'ai rajouté un petit commentaire, j'espère que c'est assez explicite sinon je peux plus détailler au besoin